parlai/core/teachers.py

```diff
@@ -2164,12 +2170,21 @@
         return output
 
     def get(self, episode_idx, entry_idx=0):
-        queue_output = self.samples.get()
-        if queue_output is None:
-            return None
+        if self.episode_done:
```

Comment:
Would prefer this private I think

Reply:
What do you mean? I actually didn't add this varialbe, I believe it's something used by FixedDialogTeacher

Reply:
we could just make this `self._episode_done` since this teacher doesn't actually use the `self.episode_done` from `FixedDialogTeacher` right? or does it?